workflow_scripts/protobuf/build_protobuf_win.ps1

```diff
@@ -24,7 +24,18 @@ mkdir build
 cd build
 $protobuf_root_dir = Get-Location
 
-cmake -G "Visual Studio 17 2022" -A $arch -Dprotobuf_MSVC_STATIC_RUNTIME=OFF -DABSL_MSVC_STATIC_RUNTIME=OFF -DBUILD_SHARED_LIBS=OFF -Dprotobuf_BUILD_TESTS=OFF -Dprotobuf_BUILD_EXAMPLES=OFF -DABSL_ROOT_DIR="$protobuf_root_dir/../../abseil-cpp-20230802.2" -DCMAKE_CXX_STANDARD=17 -DABSL_PROPAGATE_CXX_STD=on -DCMAKE_INSTALL_PREFIX="$protobuf_install_dir" ..
+# Adjust CMake generator and architecture for ARM64
+if ($arch -ieq "arm64") {
+    $cmake_arch = "ARM64"
+} elseif ($arch -ieq "x64") {
+    $cmake_arch = "x64"
+} elseif ($arch -ieq "Win32") {
+    $cmake_arch = "Win32"
+} else {
+    throw "Unsupported architecture: $arch"
+}
```

Comment:
I think this can be reverted right? As long as we provide the correct argument over in .github/workflows/release_win_aarch64.yml

Reply:
updated